### DISCUSSION TITLE
v2.3.4 user interface suggestions

### DISCUSSION THREAD

**Discussion by nippur72** [DeFiCh/app#831] [Ideas]
I just upgraded to v2.3.4, some suggestions for the user interface to make it more functional:

- Balances page
   - show more than 5 wallets per page. I have 8 wallets and they now span over two pages (before it was just one). And by the way, why are they called "wallets" in the first place, isn't that confusing with the global app wallet?
   - the arrow icons for swap, send, receive are not very intuitive... can we have it captioned ? (eg. <-> swap, ^ send, V receive)
   - the arrow icons also need "tooltips" when the mouse is hovered, suggesting what they are meant for.
   
- Left menu bar
   - add tooltips also to "lock/unlock", "cli", "help" and "settings" icons.  
   - I would also invert the color for the locked wallet: pink when it's unlocked (warning), gray when is in its normal closed state.

- DEX page   
   - do not enable the SWAP button if the wallet is locked, or at least add an "UNLOCK wallet" button side to it if the wallet is locked.
   
- Tokens page
   - add tooltip to the "verified" icon
   - tradeable "true"/"false" change it to tradeable "yes"/"no" (it's less technical); same for "minting support"

**Comment by memzila** [comment]
Yes agree on the balance page. First doesn't found my LM Token. Best allow more then 5 or make an option to hide Tokens with 0 Balance

**Comment by John-Gee** [comment]
Agreed on the DEX part, and same for LM. I always hated how if I click on it and didn't unlock yet, I'm taken back to the main page...
